sg_otio/sg_cut_track_writer.py

```diff
@@ -907,18 +924,9 @@ def _get_shot_payload(self, clip_group, sg_project, sg_linked_entity, sg_user=No
         if True:
             shot_payload[sfg.has_effects] = clip_group.has_effects
             shot_payload[sfg.has_retime] = clip_group.has_retime
-        if sfg.absolute_cut_order:
-            # TODO: maybe create Shot fields config before so that
-            #       we can get the field without an extra query?
-            sg_entity = self._sg.find_one(
-                sg_linked_entity["type"],
-                [["id", "is", sg_linked_entity["id"]]],
-                [sfg.absolute_cut_order]
-            )
-            entity_cut_order = sg_entity.get(sfg.absolute_cut_order)
-            if entity_cut_order:
-                absolute_cut_order = 1000 * entity_cut_order + clip_group.index
-                shot_payload[sfg.absolute_cut_order] = absolute_cut_order
+        if entity_cut_order:
```

Comment:
Seems strange to pass as a parameter something which is apparently already in the `sg_linked_entity` dictionary?